msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationJavaTest.java

```diff
@@ -124,6 +127,16 @@ public String getConfigFilePath() {
         return "src/test/res/raw/native_auth_native_only_test_config.json";
     }
 
+    @BeforeClass
+    public static void setupClass() {
+        BuildValues.setUseRealAuthority(true);
```

Comment:
Will use new method signature for `setUseRealAuthority` when its refactoring PR gets merged.